### PR TITLE
MTM-57429 Description of "Ignore Case on Username Login" feature

### PR DESCRIPTION
**Pull request by kowalczykmike**
The description includes a visualization of the toggle switch enabling the activation of the feature. It outlines its main functionalities, assumptions, and restrictions associated with the "Ignore Case on Username Login" feature.

### CODE REVIEW

**Review by Michal-Bak — APPROVED** [review]

**Comment by kowalczykmike** [comment]
@BeateRixen & @MWindel  can we merge it ? :) I need approval from You

File: content/authentication/basic-settings.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ By default, users can use any password with eight characters or more. If you sel
 The password validity limit and the password strength may not be editable, if configured by the platform administrator.
 {{< /c8y-admon-info >}}
 
+The **Ignore case when logging in** toggle switch allows enabling or disabling the feature, to authenticate the user into the platform without considering case sensitivity in the username or alias. The feature is applied to all tenant users when the toggle switch is set to enabled. By default, the feature is disabled, and user authentication is carried out in the standard manner.
+{{< c8y-admon-info >}}
+The toggle switch can only be managed by a user with the "tenant administrator" role. Additionally, the feature can be enabled only if there are no case-insensitive collisions for the username or alias fields for all tenant users (excluding "device users"). The check for naming collisions is performed automatically when attempting to enable the feature.
```
Comment (BeateRixen) [inline content/authentication/basic-settings.md:78]:
```suggestion
The toggle can only be managed by a tenant administrator. Additionally, the feature can only be enabled if there are no case-insensitive collisions for the username or alias fields for all existing tenant users (excluding "device users"). The check for naming collisions is performed automatically when attempting to enable the feature.
```

**Comment by BeateRixen** [comment]
@Michal-Bak The image has not the correct size. It must be 1440 px width.

**Comment by BeateRixen** [comment]
> @BeateRixen & @MWindel can we merge it ? :) I need approval from You

The PR still had a draft label which is why I did not check it earlier.

**Comment by BeateRixen** [comment]
@kowalczykmike What is the status of this PR? Can you please implement my suggestions?

**Comment by kowalczykmike** [comment]
> @kowalczykmike What is the status of this PR? Can you please implement my suggestions?

Yes, I pushed changes on Friday but forgot to add them to git before committing. 🤦‍♂️ @BeateRixen Thank you for the reminder -> now the changes should be in place. 👍

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**